benchmark/src/main/java/org/corfudb/benchmark/Common.java

```diff
@@ -0,0 +1,33 @@
+package org.corfudb.benchmark;
+
+import org.apache.commons.math3.random.BitsStreamGenerator;
+import org.apache.commons.math3.random.MersenneTwister;
+import org.corfudb.runtime.CorfuRuntime;
+import org.openjdk.jmh.annotations.Param;
+import org.openjdk.jmh.annotations.State;
+
+import static org.openjdk.jmh.annotations.Scope.Benchmark;
+
+@State(Benchmark)
+public class Common {
+    final BitsStreamGenerator RND = new MersenneTwister();
+
+    CorfuRuntime rt;
+
+    @Param("1000")
+    int num;
+
+    @Param("9000")
+    String port;
+
+    public void setup() {
+        System.out.println("stats");
+        System.out.println(port);
+        System.out.println(num);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)